packages/default/scss/dropdowngrid/_layout.scss

```diff
@@ -129,11 +133,20 @@
 
     // RTL
     .k-rtl {
-        .k-grid-list > .k-item > .k-cell {
+        &.k-dropdowngrid-popup .k-grid-header-wrap,
+        .k-dropdowngrid-popup .k-grid-header-wrap {
+            border-right-width: 0;
             border-left-width: $grid-cell-vertical-border-width;
+        }
+        &.k-dropdowngrid-popup .k-grid-header .k-header,
+        .k-dropdowngrid-popup .k-grid-header .k-header,
+        .k-grid-list > .k-item > .k-cell {
+            border-left-width: 0;
             border-right-width: $grid-cell-vertical-border-width;
             text-align: right;
         }
+        &.k-dropdowngrid-popup .k-grid-header .k-header:first-child,
+        .k-dropdowngrid-popup .k-grid-header .k-header:first-child,
```

Comment:
Is this likely to happen, having both `.k-rtl .k-dropdowngrid-popup` _and_ `.k-dropdowngrid-popup.k-rtl` ? Other than that, :shipit:

Reply:
I am not sure if we always append `k-rtl` to popups. it might be coming from the body. Hence the double selector. After R3, I will take some time to investigate.